spec/watir/rspec/helper_spec.rb

```diff
@@ -7,50 +7,52 @@
   context "#browser" do
     it "delegates to @browser if exists" do
       @browser = :browser
-      browser.should == @browser
+      expect(browser).to be @browser
     end
 
     it "delegates to @browser if @browser exists even if $browser exists" do
       @browser = :browser
       $browser = :global_browser
-      browser.should == @browser
+      expect(browser).to be @browser
     end
 
     it "delegates to $browser if @browser does not exist" do
       $browser = :global_browser
-      browser.should == $browser
+      expect(browser).to be $browser
     end
   end
 
   context "#method_missing" do
     it "redirects missing methods to browser if method exists" do
       @browser = double("browser", coolness_factor: :very)
-      coolness_factor.should == :very
+      expect(coolness_factor).to eql :very
     end
 
     it "raises error when browser does not have method" do
       @browser = double("browser")
-      described_class.should_not be_method_defined :not_existing_method
+      expect(described_class.instance_methods).not_to include :method_to_be_defined
 
       expect do
         self.not_existing_method
       end.to raise_error(NoMethodError)
 
-      described_class.should_not be_method_defined :not_existing_method
+      expect(described_class.instance_methods).not_to include :method_to_be_defined
     end
 
     it "adds browser methods to the helper" do
-      @browser = double("browser", method_to_be_defined: :done)
-      described_class.should_not be_method_defined :method_to_be_defined
-      method_to_be_defined.should == :done
-      described_class.should be_method_defined :method_to_be_defined
+      @browser = double("browser", method_to_be_defined2: :done)
```

Comment:
I'm having issues with this test when using the 

expect(described_class).to respond_to :method_to_be_defined
  and
expect(described_class).not_to respond_to :method_to_be_defined

somehow the last two tests seem to interfere unless I use the syntax as commited  + use a different method name for the two tests. I don't really understand why yet ....
I'll see if I can digg in to this next Tuesday .